.github/workflows/build-spack.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     name: build_spack
     runs-on: ubuntu-latest
     timeout-minutes: 15
-    container: precice/ci-spack-pyprecice-deps-1804
+    container: precice/ci-spack-pyprecice-deps-2004
```

Comment:
It is very strange that spack cannot detect the OS. Maybe there is something off with the image:

https://github.com/precice/python-bindings/runs/7999503469?check_suite_focus=true#step:5:99